src/java/org/apache/cassandra/service/accord/serializers/CommandsForKeySerializer.java

```diff
@@ -389,9 +389,11 @@ public static ByteBuffer toBytesWithoutKey(CommandsForKey cfk)
             {
                 Unmanaged unmanaged = cfk.getUnmanaged(i);
                 Invariants.checkState(unmanaged.pending == pending);
-                CommandSerializers.txnId.serialize(unmanaged.txnId, out, ByteBufferAccessor.instance, out.position());
+
+                CommandSerializers.txnId.serialize(unmanaged.txnId, out, ByteBufferAccessor.instance, 0);
```

Comment:
both this and and previous implementation are inconsistent with how accessor serializers are used. The pattern is:

```java
long position = 0;
position += CommandSerializers.txnId.serialize(unmanaged.txnId, out, ByteBufferAccessor.instance, position);
position += CommandSerializers.timestamp.serialize(unmanaged.waitingUntil, out, ByteBufferAccessor.instance, position);
return position;
```